packages/untp-test-suite/src/core/utils/common.ts

```diff
@@ -29,6 +15,13 @@ export const readFile = async <T>(filePath: string): Promise<T> => {
   return JSON.parse(fileContent);
 };
 
+export const readConfigFile = async () => {
```

Comment:
If we add a parameter to specify the path of the credential configuration file we want to read, I think it should be passed to this function. Since this function is primarily responsible for reading the credential configuration file.